openapi/openapi.yaml

```diff
@@ -2,7 +2,7 @@ openapi: "3.0.0"
 info:
   title: "ExplainaBoard"
   description: "Backend APIs for ExplainaBoard"
-  version: "0.2.21"
+  version: "0.2.22"
```

Comment:
This version number change will clash with https://github.com/neulab/explainaboard_web/pull/539

Reply:
Thanks for the reminder. I'll change the version number before merge.